app/scripts/controllers/replicaSet.js

```diff
@@ -79,9 +78,10 @@ angular.module('openshiftConsole')
 
     $scope.deploymentsVersion = APIService.getPreferredVersion('deployments');
     $scope.deploymentConfigsVersion = APIService.getPreferredVersion('deploymentconfigs');
+    $scope.deploymentConfigsLogVersion = APIService.getPreferredVersion('deploymentconfigs/log');
     $scope.eventsVersion = APIService.getPreferredVersion('events');
-    // TODO: update common/constants/apiPreferredVersions for this
-    $scope.deploymentConfigsLogVersion = 'deploymentconfigs/log';
+    $scope.horizontalPodAutoscalersVersion = APIService.getPreferredVersion('horizontalpodautoscalers');
+    $scope.replicationControllersVersion = replicationControllersVersion;
```

Comment:
Assign the value directly to the scope var and use the scope var in the controller if needed to be consistent with other preferred versions

Reply:
Can do, I did it this way because the value is used in a switch statement above already.  I can just move this whole block up.